### ISSUE TITLE
`inline-asm` on nightly

### ISSUE THREAD

**Issue by little-arhat** [rust-embedded/cortex-m#228]
Hello,

With new `asm!` macro merged, it's impossible to use `cotrex-m` crate with `inline-asm` on nightly, because latest released version (0.6.2) still uses old `asm!`. 
At the same time it's impossible to use cortex-m from git, because it then conflicts with `cortex-m` brough by other deps (`rtfm`, `cortex-m-log`, etc):

```
error: failed to select a version for `cortex-m`.
    ... required by package `cortex-m-log v0.6.1`
    ... which is depended on by `fcfs-rtfm v0.1.0 (/Users/r/prj/p/copterust/fcfs-rtfm)`
versions that meet the requirements `= 0.6.2` are: 0.6.2

the package `cortex-m` links to the native library `cortex-m`, but it conflicts with a previous package which links to `cortex-m` as well:
package `cortex-m v0.6.2 (https://github.com/rust-embedded/cortex-m?rev=c9c7539233954822a6132f4bc13e5763371b5cb2#c9c75392)`
    ... which is depended on by `fcfs-rtfm v0.1.0 (/Users/r/prj/p/copterust/fcfs-rtfm)`
```

What's the recommended course of action, besides not using `inline-asm`?  (refs #217).

**Comment by adamgreig** [comment]
@little-arhat did using patch work OK for you?

**Comment by little-arhat** [comment]
@adamgreig yes, thanks! is there ETA for a proper fix?

**Comment by adamgreig** [comment]
The next release will have a fix; see #217 for updates but hopefully soon.
